Doc/library/socket.rst

```diff
@@ -408,6 +408,9 @@ Constants
    in the Unix header files are defined; for a few symbols, default values are
    provided.
 
+   .. versionchanged:: 3.13 On Windows, ``TCP_QUICKACK`` appear if run-time Windows
+      supports.
```

Comment:
Put this entry at the end of the versionchanged list.  Btw, does it work normally on linux? (namely, you don't need special treatment for this constant or?)

Reply:
Linux does define TCP_QUICKACK so that define section works as normal without any special remap like I had to do for Windows.  The logic for implementation on Linux is unchanged / unaffected.  I did try it out to confirm support.

Oops, thought the versions were going the other way.  Moving to the end of the list.

Reply:
>  The logic for implementation on Linux is unchanged / unaffected. I did try it out to confirm support.

Then you can say "Added support for ``TCP_QUICKACK`` on Windows platforms when available."

Reply:
Sure, I'll update the phrasing of the other version notes in this section as well.  I agree this phrasing is not ideal, I was just copying what was already there.  For example: 

   .. versionchanged:: 3.6.5
      On Windows, ``TCP_FASTOPEN``, ``TCP_KEEPCNT`` appear if run-time Windows
      supports.

Since I'm making the phrasing on this one better, might as well do so for the rest, just for consistency.

Reply:
In general it's better to keep old code but I agree that the old formulations are not the best ones..